app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -1093,4 +1083,131 @@ private void shareMessage(String message) {
             currContext.startActivity(sendIntent);
         }
     }
+
+    private class SearchResultsAdapter extends RecyclerView.Adapter<SearchResultHolder> {
```

Comment:
Actually I could not get the reference of Action mode callback in other class. I am looking into it. If you can suggest a better implementation than this by which I could control the menu action from other class it would be very helpful.
Thanks